docs/reference/run.md

```diff
@@ -1385,6 +1385,17 @@ Similarly the operator can set the **hostname** with `-h`.
 > **Note**:
 > The auto-creation of the host path has been [*deprecated*](../misc/deprecated.md#auto-creating-missing-host-paths-for-bind-mounts).
 
+
+> **Note**:
+> When using systemd to manage the Docker daemon's start and stop, in the systemd
+> unit file there is an option to control mount propagation for the Docker daemon
+> itself, called `MountFlags`. The value of this setting may cause Docker to not
+> see mount propagation changes made on the mount point. For example, if this value
+> is `slave`, you may not be able to use the `shared` or `rshared` propagation on
+> a volume.
+
+
+
```

Comment:
why so many returns jeez its like you are running linux and used too much pressure :smiling_imp: